app/views/houses/edit.html.slim

```diff
@@ -15,7 +15,7 @@
           .col-9 = h.text_field :house_name
         .form-group.row
           .col-3 = label :reason, "Reason:"
-          .col-9 = text_field :reason, :reason
+          .col-9 = text_field :reason, :reason, required: true

```

Comment:
In the controller you made it *no longer* required, but you're making it required in the form?

Reply:
readded to controller